packages/component-library/draft/Kaizen/Modal/presets/ConfirmationModal.tsx

```diff
@@ -20,7 +22,7 @@ const styles = require("./ConfirmationModal.scss")
 
 interface Props {
   readonly isOpen: boolean
-  readonly type: "positive" | "informative" | "negative"
+  readonly type: ModalType
```

Comment:
This is a nitpick, but calling this `Mood` would make the language consistent with how the designers talk about modals.

Reply:
(I see you didn't introduce the word `type`, but it could be a good improvement)